src/commands/latency-latest.json

```
@@ -20,11 +20,11 @@
         ],
         "reply_schema": {
             "type": "array",
-            "description": "An array where each element is a four elements array representing the event's name, timestamp, latest and all-time latency measurements.",
+            "description": "An array where each element is a n elements array representing the event name and information about the event latency.",
```

"n elements" doesn't mean much if we don't say what *n* is. We can just say *an array*?

"about the event latency" also doesn't say much. It could be also about EVENT HISTORY. I think "timestamp, latest and all-time latency measurements" is OK. Maximum, sum and count can all be "all-time measurements", right?

```suggestion
            "description": "An array where each element is an array representing the event name, timestamp, latest and all-time latency measurements.",
```